### PR TITLE
feat: add mdformat

### PR DESCRIPTION
**Pull request by anand-bala**
[`mdformat`](https://github.com/executablebooks/mdformat) is a Markdown formatter written in Python that is extensible using plugins.

### CODE REVIEW

**Review by williamboman — APPROVED** [review]
Thanks!

File: packages/mdformat/package.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: mdformat
+description: CommonMark compliant Markdown formatter
+
```
Comment (williamboman) [inline packages/mdformat/package.yaml:4]:
```suggestion
```

File: packages/mdformat/package.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+name: mdformat
+description: CommonMark compliant Markdown formatter
```
Comment (williamboman) [inline packages/mdformat/package.yaml:3]:
```suggestion
description: CommonMark compliant Markdown formatter.
```

**Comment by tnthi115** [comment]
Is it possible to add [`mdformat-gfm`](https://pypi.org/project/mdformat-gfm/) and other options listed [here](https://pypi.org/project/mdformat-gfm/)?

**Pull request merged**